src/applications/caregivers/tests/components/ConfirmationPage/ConfirmationScreenView.unit.spec.js

```diff
@@ -1,41 +1,71 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { expect } from 'chai';
-import moment from 'moment';
```

Comment:
Are we removing this because when don't need timestamps anymore or is it to improve test perfomance?

Reply:
The expect that contained the moment use was redundant to the expect with the actual date string. Combine that with the fact that moment is actually a depricated library, I try to really limit its use. So choosing to eliminate it here.